src/bitmessageqt/languagebox.py

```diff
@@ -8,24 +10,27 @@
 
 
 class LanguageBox(QtGui.QComboBox):
+    """LanguageBox class for QT UI"""
     languageName = {
         "system": "System Settings", "eo": "Esperanto",
         "en_pirate": "Pirate English"
     }
 
     def __init__(self, parent=None):
+        """Initialization"""
         super(QtGui.QComboBox, self).__init__(parent)
         self.populate()
 
     def populate(self):
+        """Populate comboBox based on languageName"""
```

Comment:
Wrong. It populates drop down list with all available languages.

Reply:
@g1itch done.